tss-esapi-sys/.cargo/config

```diff
@@ -0,0 +1,7 @@
+# Cargo seems to expect the -gcc variant instead of -ld
+
+[target.armv7-unknown-linux-gnueabi]
+linker = "arm-linux-gnueabihf-gcc"
```

Comment:
Maybe it should use `arm-linux-gnueabi-gcc`, without the `hf`, if that exists.

Reply:
Removed it completely.